bilby/gw/likelihood.py

```diff
@@ -1628,22 +1629,22 @@ def compute_summary_data(self):
                 interferometer.frequency_domain_strain[mask],
                 self.per_detector_fiducial_waveforms[interferometer.name][mask],
                 interferometer.power_spectral_density_array[mask],
-                self.waveform_generator.duration)
+                self.waveform_generator.duration) / self.number_of_bins
             b0 = noise_weighted_inner_product(
                 self.per_detector_fiducial_waveforms[interferometer.name][mask],
                 self.per_detector_fiducial_waveforms[interferometer.name][mask],
                 interferometer.power_spectral_density_array[mask],
-                self.waveform_generator.duration)
+                self.waveform_generator.duration) / self.number_of_bins
             a1 = noise_weighted_inner_product(
                 interferometer.frequency_domain_strain[mask] * factor,
```

Comment:
I don't know what the variable `factor` is here. I got the same likelihood values as before even after removing it.

Reply:
The formula for `a1` is:

 <img src="https://render.githubusercontent.com/render/math?math=A_1(b)=4\sum \frac{[d(f) \times (f-f_m(b) ] h^* (f)}{S_n(f)} )">

`factor` then is just the `f-f_m(b)`.

Reply:
But please check this explicitly.

 The `a1` and `b1` values calculated for `init_params` won't change the likelihood calculation since the waveform ratio is 1 and hence `r1` is zero.

Reply:
That clears things up. But I still don't know where this factor of `#bins` is coming from. I'll do some checks.